example/lambdas/python-processing/package.json

```diff
@@ -10,10 +10,12 @@
   },
   "scripts": {
     "test": "python -m unittest",
-    "python-lint": "pylint *.py",
+    "python-lint": "pipenv run pylint *.py",
     "lint": "npm run python-lint",
-    "build": "pip install -r requirements-dev.txt && pip install -r requirements.txt",
-    "package": "npm run build",
+    "clean": "rm -rf .venv",
+    "build": "pip install pipenv && PIPENV_VENV_IN_PROJECT=1 pipenv install --dev --deploy --ignore-pipfile",
+    "prepare": "npm run build",
+    "package": "true",
     "install-python-deps": "pip install -r requirements-dev.txt && pip install -r requirements.txt"
```

Comment:
🤔  Should we be running this at all and/or should we be using the pipenvfor the unit tests/lint?

Reply:
https://github.com/nasa/cumulus/pull/2038/commits/3ca10b1247a4967b9874e3c8c8c99bf0fb7ec700 👍